template/CHART_NAME/templates/NOTES.txt

```diff
@@ -32,6 +32,12 @@ In order to replicate the container startup scripts execute this command:
 
 {{- include "common.warnings.rollingTag" .Values.%%MAIN_OBJECT_BLOCK%%.image }}
 {{- include "common.warnings.rollingTag" .Values.%%OTHER_OBJECT_BLOCK%%.image }}
+{{- include "common.warnings.rollingTag" .Values.defaultInitContainers.volumePermissions.image }}
+{{- $resourcesSections := list "%%MAIN_OBJECT_BLOCK%%" "%%OTHER_OBJECT_BLOCK%%." }}
+{{- if .Values.defaultInitContainers.volumePermissions.enabled }}
```

Comment:
If I recall correctly, the common.warning.resources checks if a section has the `enabled` value and if it is set to true

Reply:
Done at https://github.com/bitnami/charts/pull/34344/commits/2a9f150df24228b57a853c9692d4d7eaf5131c2d